starter_code/index.html

```diff
@@ -44,19 +49,19 @@
       <form id="new-character-form" class="character-form">
         <div class="field">
           <label for="name">Name: </label>
-          <input type="text" name="name">
```

Comment:
You don't have to use an `id`. You can instead use a selector that matches on the `name` attribute.
`id`s are convenient to use but they have a big downside: you can only have one of each. Preventing name clashes is hard, so just use `id`s for what they're good for: anchors!